### ISSUE TITLE
Apply learning rate scheduler not to PPO

### ISSUE THREAD

**Issue by timmeinhardt** [ikostrikov/pytorch-a2c-ppo-acktr#155]
In contrast to A2C and A2C_ACKTR, the PPO implementation already includes a learning rate scheduling performed by Adam. In supervised learning it is debatable if one should use manual scheduling in combination with Adam, but in doubt I would recommend to not interfere with the Adam intrinsic scheduling. 

Maybe it makes sense to investigate this issue in RL but until then not to activate the linear scheduling for PPO by default.

**Comment by ikostrikov** [comment]
I did it mostly to match the baselines setup. I don't have enough resources to run a massive hyper parameter search myself and I assume that they compare results with and without decaying learning rates.

**Comment by timmeinhardt** [comment]
Matching the baseline setup is of course a good argument. Maybe they thought about this. But it is definitely something that one should keep in mind. 

I will post the same issue to baselines and see if they can elaborate on this.
